### ISSUE TITLE
Improve the logs for healthcheck steps

### ISSUE THREAD

**Issue by jeawhanlee** [wp-media/wp-rocket-e2e#118]
**Is your feature request related to a problem? Please describe.**
The logs on the CLI for the healthcheck steps are not very detailed with validations to check when certain blocks fail.

**Describe the solution you'd like**
We should validate each block carefully and log detailed info to the CLI. We are already doing this for the [checkWPStatus](https://github.com/wp-media/wp-rocket-e2e/blob/e43c0ea4fab6c17ef930ba8f7f46f5ddf94a1078/healthcheck.ts#L9) but not for the rest of the steps that involve setting up the browser and visiting the page. 
In summary, we should clearly log the start of a step and in between if it fails or not and then the end eventually.

**Describe alternatives you've considered**

**Additional context**
https://github.com/wp-media/wp-rocket-e2e/pull/112
